### PR TITLE
docs(android-dev-env): improve docs

### PR DESCRIPTION
**Pull request by Simon-TechForm**
## Description

I recently went through the `Setting Up an Android Development & Testing Environment` guide and came across a couple of things that I would've liked was included in the doc.

Check out the commit descriptions for the reasoning.

### CODE REVIEW

File: docs/guide/android-dev-env.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ _Here’s a visual comparison between the two - an SDK 28 (Android 9) AOSP emula
 
 While it’s possible to do this using Android Studio, we’ll focus on the command line, as it is also good for _headless_ CI machines.
 
-1. Locate your 'Android home' folder - typically set in the `ANDROID_HOME` environment variable, or in its successor - `ANDROID_SDK_ROOT`. If `ANDROID_HOME` isn’t set, either set it yourself or run the following commands after `cd`-ing into the home folder.
+1. Locate your 'Android home' folder - typically set in the `ANDROID_HOME` environment variable, or in its predecessor - `ANDROID_SDK_ROOT`. If `ANDROID_HOME` isn’t set, either set it yourself or run the following commands after `cd`-ing into the home folder.
```
Comment (asafkorem) [inline docs/guide/android-dev-env.md:81]: Right, nice catch!

File: docs/guide/android-dev-env.md [hunk 2]
```diff
@@ -92,6 +92,8 @@ While it’s possible to do this using Android Studio, we’ll focus on the comm
    In previous Android Studio versions, the SDK-Manager path was located at `$ANDROID_HOME/tools/bin/sdkmanager`.
    We highly recommend working with the latest version, however, in case you are working with an old version, and this command fails, try this path instead.
 
+   The command might also fail if the command-line tools aren't installed. To install the tools using Android Studio, click `Tools > SDK Manager` and select the `SDK Tools` tab. Ensure that `Android SDK Command-line Tools (latest)` is checked and applied.
```
Comment (asafkorem) [inline docs/guide/android-dev-env.md:95]: 💯

**Review by asafkorem — APPROVED** [review]
Love it! Thanks for your contribution @Simon-TechForm

File: docs/guide/android-dev-env.md (hunk 1 above)
Comment (d4vidi) [reply docs/guide/android-dev-env.md:81]: Oh wow. At the time writing this, I was genuinely under the impression that `ANDROID_HOME` was the one they've deprecated. Thanks again!

File: docs/guide/android-dev-env.md [hunk 3]
```diff
@@ -104,6 +106,7 @@ While it’s possible to do this using Android Studio, we’ll focus on the comm
    > - With `;android-28;`, we assumed SDK 28 here, but other APIs are supported just the same.
    > - The `;default;` part replaces `;google_apis;`, which is the default, and is what matters here.
    > - If you are running on a M1 you must install an arm64 architecture i.e. `system-images;android-28;default;arm64-v8a`
+   > - To get a list of available images, run `$ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --list`
```
Comment (d4vidi) [inline docs/guide/android-dev-env.md:109]: I'm not at all confident about having `cmdline-tools` so deterministically set up in all environments (my doesn't have it...). I think the safer / alternate path to point out is `$ANDROID_HOME/tools/bin/sdkmanager`.

**Pull request merged**